src/module/entities/helper.js

```diff
@@ -386,8 +387,14 @@ module.exports = class UserProjectsHelper {
 					},
 					status: CONSTANTS.common.ACTIVE_STATUS,
 				}
+
+				let userRoleExtensionProjection
+				if (!language || language == 'en') {
```

Comment:
@MallanagoudaB instead of using 'en' define a common variable ENGLISH_LANGUGE_CODE  and use it every where we used 'en'